jit/v2/JIT_v2.0.0.yaml

```diff
@@ -174,30 +181,139 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-  '/v2/port-call-services':
-    get:
+  '/v2/port-calls/{portCallID}/omit':
+    post:
       parameters:
-        - $ref: '#/components/parameters/portCallIDQueryParam'
-        - $ref: '#/components/parameters/terminalCallIDQueryParam'
-        - $ref: '#/components/parameters/portCallServiceIDQueryParam'
+        - $ref: '#/components/parameters/portCallIDPathParam'
+        - $ref: '#/components/parameters/Api-Version-Major'
+      tags:
+        - Port Call Service consumer
```

Comment:
'C'onsumer? -- just that it looks aligned with Port Call Service

Reply:
Modified it here plus all other places to: **Port Call Service - Consumer** (same for Provider tags)